### ISSUE TITLE
background image 

### ISSUE THREAD

**Issue by InnayTool** [foss-ag/website_hack-and-snack#3]
The background image of the page should be individualised.

**Comment by dragetd** [comment]
I agree! But if I try to draw something, I'll get rotten food thrown at me next meeting. ;-P So feel free to suggest graphics! :)

**Comment by ROYALBEFF** [comment]
I suggest to use a variation of the banner we already use on [twitter](https://twitter.com/FOSSAGTUDO). Maybe we could simply remove the writing and only use the coloured grid.

**Comment by Lazalatin** [comment]
I would really like to see @ROYALBEFF s approach!
